Dockerfile

```diff
@@ -11,12 +11,18 @@ ENV ADMIN_PORT=3005
 EXPOSE ${HTTP_PORT}/tcp ${WS_PORT}/tcp ${CDN_PORT}/tcp ${RTC_PORT}/tcp ${ADMIN_PORT}/tcp
 
 # install required apps
-RUN apk add --no-cache --update git python3 py-pip make build-base
+RUN apk add --no-cache --update git python3 py-pip make build-base g++ pkgconfig pixman-dev cairo-dev pango-dev
 RUN ln -s /usr/bin/python3 /usr/bin/python
+RUN npm install --global patch-package --save
```

Comment:
Why do you install `patch-package` globally? It's already included in the `package.json` deps.